src/Lunar.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Lunar;
+
+use Lunar\Endpoint\Payments;
+use Lunar\HttpClient\CurlClient;
+use Lunar\HttpClient\HttpClientInterface;
+
+/**
+ * Class Lunar
+ *
+ * @package Lunar
+ */
+class Lunar
+{
+	/**
+	 * @var string
+	 */
+	const BASE_URL = 'https://api.prod.lunarway.com/merchant-payments/v1';
+	const TEST_BASE_URL = 'https://api.dev.lunarway.com/merchant-payments/v1';
+
+	/**
+	 * @var HttpClientInterface
+	 */
+	public $client;
+
+	/**
+	 * @var string
+	 */
+	private $api_key;
+
+	private $version = '1.0.0';
```

Comment:
This is current version? It is different from composer.json file. I think this is the correct version.